docs/man-pages/condor_suspend.rst

```diff
@@ -1,76 +1,102 @@
-      
-
 *condor_suspend*
-=================
+================
 
-suspend jobs from the HTCondor queue
-:index:`condor_suspend<single: condor_suspend; HTCondor commands>`\ :index:`condor_suspend command`
+Suspend running job(s) from the HTCondor queue.
+:index:`condor_suspend<double: condor_suspend; HTCondor commands>`
 
 Synopsis
 --------
 
-**condor_suspend** [**-help | -version** ]
+**condor_suspend** [**-help** | **-version**]
+
+**condor_suspend** [*OPTIONS*] [*cluster*... | *cluster.proc*... | *user*...]
 
-**condor_suspend** [**-debug** ] [
-**-pool** *centralmanagerhostname[:portnumber]* |
-**-name** *scheddname* ] | [**-addr** *"<a.b.c.d:port>"*] **
+**condor_suspend** [**-debug**] [**-long**] [**-totals**] [**-all**] [**-constraint** *expression*]
+[**-pool** *hostname[:portnumber]* | **-name** *scheddname* | **-addr** *"<a.b.c.d:port>"*]
 
 Description
 -----------
 
-*condor_suspend* suspends one or more jobs from the HTCondor job queue.
-When a job is suspended, the match between the *condor_schedd* and
-machine is not been broken, such that the claim is still valid. But, the
-job is not making any progress and HTCondor is no longer generating a
-load on the machine. If the **-name** option is specified, the named
-*condor_schedd* is targeted for processing. Otherwise, the local
-*condor_schedd* is targeted. The job(s) to be suspended are identified
-by one of the job identifiers, as described below. For any given job,
-only the owner of the job or one of the queue super users (defined by
-the :macro:`QUEUE_SUPER_USERS` macro) can suspend the job.
+*condor_suspend* suspends one or more running jobs managed by an Access
+Point. This command will result in the job process being suspended on the
+Execution Point thus not making progress nor generating load on the machine.
```

Comment:
Let's remove the phrase "thus not making progress nor generating load on the machine".

Instead, let's start a new sentence defining what suspended jobs are.

"All the processes in a suspended job are sent the SIGSTOP signal, or equivalent.  They consume no cpu usage, but continue to use memory and scratch disk space.  Suspended jobs still consume the slot they run in. Suspended jobs are still charged to submitters in term of user priority.  An administrator might want to suspend on the jobs in a pool to quickly reduce the power draw in a pool, in the case where they may be able to :tool:`condor_continue` the jobs in a short while, and do not wish the jobs to be vacated, and then restart from their beginning or last checkpoint.